### PR TITLE
bento: feat: bento cli

### PR DESCRIPTION
**Pull request by bidah**

### CODE REVIEW

File: packages/bento-cli/package.json
Comment (natew) [inline packages/bento-cli/package.json:1]: 2 space indent

File: packages/bento-cli/source/commands/add.tsx
Comment (natew) [inline packages/bento-cli/source/commands/add.tsx:1]: 2 space indents everywhere

File: packages/bento-cli/source/components.ts
Comment (natew) [inline packages/bento-cli/source/components.ts:1]: is this used anywhere?

File: packages/bento-cli/source/constants.ts
Comment (natew) [inline packages/bento-cli/source/constants.ts:1]: duplicate github client id

File: packages/bento-cli/source/commands/counter.tsx
Comment (natew) [inline packages/bento-cli/source/commands/counter.tsx:1]: is this a relevant file?

File: packages/bento-cli/source/components.ts
Comment (bidah) [reply packages/bento-cli/source/components.ts:1]: yes its the schema for all searches of components

File: packages/bento-cli/source/commands/test.tsx
[the remark this answers is not in the document]
Comment (bidah) [reply packages/bento-cli/source/commands/test.tsx:1]: came as the default setup of INK but I can change it 👍

File: packages/bento-cli/source/commands/counter.tsx
Comment (bidah) [reply packages/bento-cli/source/commands/counter.tsx:1]: no. was testing INK. removing.

**Pull request closed without merging**